server/homebrew.api.js

```diff
@@ -87,8 +87,18 @@ const api = {
 			// Get relevant IDs for the brew
 			const { id, googleId } = api.getId(req);
 
+			const accessMap = {
+				edit  : { editId: id },
+				share : { shareId: id },
+				admin : {
+					$or : [
+						{ editId: { $regex: id, $options: 'i' } },
+						{ shareId: { $regex: id, $options: 'i' } },
```

Comment:
Why do we need regex for this?

Reply:
I was copying the existing functionality from the Brew Lookup function, which used regex for case-insensitive search... I think we can probably skip that; I suspect that anyone with Admin access is probably going to assume that the search is case-sensitive.

Reply:
Regex searching removed in latest commit.